build-tools/unix-distribution-setup/unix-distribution-setup.mdproj

```diff
@@ -0,0 +1,26 @@
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ItemType>GenericProject</ItemType>
+    <ProjectGuid>{8FF78EB6-6FC8-46A7-8A15-EBBA9045C5FA}</ProjectGuid>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)'=='Debug'">
+    <OutputPath>..\..\bin\Debug\</OutputPath>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)'=='Release'">
+    <OutputPath>..\..\bin\Release\</OutputPath>
+  </PropertyGroup>
+
+  <Import Project="..\..\Configuration.props" />
+
+  <Target Name="Build">
+    <Copy SourceFiles="..\..\tools\scripts\generator" DestinationFiles="$(OutputPath)bin\generator" />
```

Comment:
git should, yes, but this might not just be a matter of "using bad clients".

An alternate scenario: Dropbox-based git checkout, *originally* checked out on Windows (i.e. no `+x` support). If a Linux/OSX machine is attached to the same Dropbox account, will those file permissions be preserved, or will they have been "lost" because Dropbox cares not about git, it just synchronizes files, and Windows was the first one to synchronize, so that information is lost...

Yes, we could say that's not a support scenario, but (1) that's a *really convenient* scenario (easy source sharing), and (2) it's not *that* hard to support by having an explicit `chmod +x` after copying the file.